### DISCUSSION TITLE
标题不重要的吧

### DISCUSSION THREAD

**Discussion by Chlorie** [Hoshikawa-Kaguya/Sora#91] [ワイビバボー]
内容是不是也不重要.jpg

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus lobortis, erat nec scelerisque faucibus, mi ante sodales tortor, ut condimentum quam ipsum nec nisi. Duis laoreet est a blandit sollicitudin. Sed sagittis orci eget justo dignissim, a tincidunt diam iaculis. Pellentesque tortor nisl, lacinia eget maximus a, efficitur sit amet turpis. Sed elementum mauris arcu, rhoncus accumsan leo volutpat eu. Nulla facilisi. Nunc sed sapien mattis, iaculis felis id, porta est. Proin vestibulum dolor condimentum ex vestibulum convallis non et diam. Sed ullamcorper quam id leo porttitor, vel euismod nisi luctus. Phasellus pulvinar magna non tincidunt aliquet. Vivamus fermentum tellus sit amet neque sagittis, et egestas urna interdum. Sed et iaculis justo. Aliquam erat volutpat.

**Comment by Yukari316** [comment]
草，我才看到，但好像没有入群申请？

**Reply by Chlorie** [reply]
草我还以为就在这申请（（（
